packages/editor/src/components/visual-editor/edit-template-blocks-notification.js

```diff
@@ -27,16 +28,21 @@ import { store as editorStore } from '../../store';
  *                                                                  editor iframe canvas.
  */
 export default function EditTemplateBlocksNotification( { contentRef } ) {
-	const { onNavigateToEntityRecord, templateId } = useSelect( ( select ) => {
-		const { getEditorSettings, getCurrentTemplateId } =
-			select( editorStore );
+	const { onNavigateToEntityRecord, templateId, editorMode } = useSelect(
+		( select ) => {
+			const { getEditorSettings, getCurrentTemplateId } =
+				select( editorStore );
+			const { __unstableGetEditorMode } = select( blockEditorStore );
 
-		return {
-			onNavigateToEntityRecord:
-				getEditorSettings().onNavigateToEntityRecord,
-			templateId: getCurrentTemplateId(),
-		};
-	}, [] );
+			return {
+				onNavigateToEntityRecord:
+					getEditorSettings().onNavigateToEntityRecord,
+				templateId: getCurrentTemplateId(),
+				editorMode: 'zoom-out' === __unstableGetEditorMode(),
```

Comment:
This looks like it's bettter named `isZoomOutMode`

Reply:
Changed to `isZoomOutMode`